test/greet.test.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const assert = require('assert');
+const greet = require('../lib/greet');
+
+const name = 'steven';
+
+describe('greet.js', function(){
+  describe('#greet', function(){
+    it('is a string', function(){
+      assert.equal(typeof greet.greet(name), 'string');
+    });
+    it('recieves the argument', function(){
+      assert.strictEqual(greet.greet(name), 'Hello steven');
```

Comment:
Did you find the difference between strictEqual/deepEqual, etc? Nice job. What else can you test for? This is great practice for later in the class. There will be times where TDD can help code defensively against incorrect parameters being passed into input fields.